pageComponents/NewPostPage/PlayerHandSelectors.tsx

```diff
@@ -0,0 +1,117 @@
+import * as React from "react";
+import styled from "styled-components";
+import SelectablePlayingCard from "../../components/SelectablePlayingCard";
+import { MOBILE_MEDIA } from "../../constants/mediaquery";
+import PlayingCard, { Rank, Suit } from "../../models/PlayingCard";
+import getPositionByPlayerAndIndex from "../../utilities/getPositionByPlayerAndIndex";
+
+interface Props extends React.Attributes {
+  players: {
+    index: number;
```

Comment:
I'm sorry, let me make sure. What does `index` mean? Are `players[i]` and `index` different?

Reply:
It is a PlayerIndex the is still active at the Showdown phase.

Reply:
How are they different?